server/service/src/sync/translation_and_integration.rs

```diff
@@ -160,7 +140,35 @@ impl<'a> TranslationAndIntegration<'a> {
                     );
                 }
             }
+
+            if true == initialisation {
```

Comment:
Nit: initialisation is a boolean, ya don't need to compare it!

Nit within a nit: https://en.wikipedia.org/wiki/Yoda_conditions. That said there isn't really a disadvantage it just reads a bit funny in English, and arguably there _are_ advantages as stated on wikipedia! I didn't know WordPress actually states it as a convention!